<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeddingField</class>
 <widget class="QDialog" name="WeddingField">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>234</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Bride Information</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>153</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldEmailBride">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>153</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldPasswordBride">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>127</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>127</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>101</y>
      <width>52</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldUserNameBride">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>101</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldDOBBride">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>75</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DOB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldLastNameBride">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>49</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldFirstNameBride">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>23</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Bride's Parents</string>
    </property>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dad First</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mom First</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Last</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>DOB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Last</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>DOB</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>133</width>
       <height>60</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLineEdit" name="fieldMotherBrideFN"/>
     <widget class="QLineEdit" name="fieldMotherBrideLN"/>
     <widget class="QLineEdit" name="fieldMotherBrideDOB"/>
    </widget>
    <widget class="QSplitter" name="splitter_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>133</width>
       <height>60</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLineEdit" name="fieldFatherBrideFN"/>
     <widget class="QLineEdit" name="fieldFatherBrideLN"/>
     <widget class="QLineEdit" name="fieldFatherBrideDOB"/>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldPhoneBride">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>180</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>251</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>Wedding Information</string>
   </property>
   <widget class="QGroupBox" name="grpBxNewBrideAndGroom">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>212</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>New Bride and Groom</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLineEdit" name="fieldWedDate"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnNewOfficial">
       <property name="text">
        <string>New Official</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Official ID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="fieldOfficialID"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="grpBxExistBrideAndGroom">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Existing Bride and or/ Groom</string>
    </property>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bride's ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Groom's ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>New Official</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Existing Official</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>190</width>
      <height>17</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QRadioButton" name="radioButtonExistBride">
     <property name="text">
      <string>Existing Bride</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonExistGroom">
     <property name="text">
      <string>Existing Groom</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>410</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>241</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Groom Information</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldFirstNameGroom">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>23</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldLastNameGroom">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>49</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DOB</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldDOBGroom">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>75</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>101</y>
      <width>52</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldUserNameGroom">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>101</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>127</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldPasswordGroom">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>127</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>153</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldEmailGroom">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>153</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Groom's Parents</string>
    </property>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dad First</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mom First</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Last</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>DOB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Last</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>DOB</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>133</width>
       <height>60</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLineEdit" name="fieldMotherGroomFN"/>
     <widget class="QLineEdit" name="fieldMotherGroomLN"/>
     <widget class="QLineEdit" name="fieldMotherGroomDOB"/>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>133</width>
       <height>60</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLineEdit" name="fieldFatherGroomFN"/>
     <widget class="QLineEdit" name="fieldFatherGroomLN"/>
     <widget class="QLineEdit" name="fieldFatherGroomDOB"/>
    </widget>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldPhoneGroom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>fieldFirstNameBride</tabstop>
  <tabstop>fieldLastNameBride</tabstop>
  <tabstop>fieldDOBBride</tabstop>
  <tabstop>fieldUserNameBride</tabstop>
  <tabstop>fieldPasswordBride</tabstop>
  <tabstop>fieldEmailBride</tabstop>
  <tabstop>fieldPhoneBride</tabstop>
  <tabstop>fieldMotherBrideFN</tabstop>
  <tabstop>fieldMotherBrideLN</tabstop>
  <tabstop>fieldMotherBrideDOB</tabstop>
  <tabstop>fieldFatherBrideFN</tabstop>
  <tabstop>fieldFatherBrideLN</tabstop>
  <tabstop>fieldFatherBrideDOB</tabstop>
  <tabstop>fieldFirstNameGroom</tabstop>
  <tabstop>fieldLastNameGroom</tabstop>
  <tabstop>fieldDOBGroom</tabstop>
  <tabstop>fieldUserNameGroom</tabstop>
  <tabstop>fieldPasswordGroom</tabstop>
  <tabstop>fieldEmailGroom</tabstop>
  <tabstop>fieldPhoneGroom</tabstop>
  <tabstop>fieldMotherGroomFN</tabstop>
  <tabstop>fieldMotherGroomLN</tabstop>
  <tabstop>fieldMotherGroomDOB</tabstop>
  <tabstop>fieldFatherGroomFN</tabstop>
  <tabstop>fieldFatherGroomLN</tabstop>
  <tabstop>fieldFatherGroomDOB</tabstop>
  <tabstop>fieldWedDate</tabstop>
  <tabstop>fieldOfficialID</tabstop>
  <tabstop>btnNewOfficial</tabstop>
  <tabstop>radioButtonExistBride</tabstop>
  <tabstop>radioButtonExistGroom</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
